from-dump/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quetzacoalt/docker-internal-images:1.7
```

Comment:
quetzacoalt/docker-internal-images:1.7 
is based on
prestashop/prestashop:1.6
Shouldn't this docker-internal-images be renamed to something like prestashopWithMysql  and be move to prestashop ?

And in the end we don't use the PrestaShop code that comes with those images but the one in the zip.